gradle/libs.versions.toml

```diff
@@ -61,12 +62,14 @@ koin-core = { module = "io.insert-koin:koin-core" }
 koin-test = { module = "io.insert-koin:koin-test" }
 kotlin-test = { module = "org.jetbrains.kotlin:kotlin-test", version.ref = "kotlin" }
 koin-junit4 = { module = "io.insert-koin:koin-test-junit4" }
+kotlinpoet = { module = "com.squareup:kotlinpoet", version.ref = "kotlinpoet" }
```

Comment:
[KotlinPoet](https://square.github.io/kotlinpoet/) is a library for generating Kotlin code; it makes some string escaping stuff a little less confusing. I tried adopting it in `buildSrc/` for our existing code generation, but it wound up being a net increase in lines of code, and the output isn’t nicely formatted so it’d be actually kind of annoying to try to use in `buildSrc/` for the things that run automatically and produce outputs that we keep in-tree.